### DISCUSSION TITLE
Empty Group Tags

### DISCUSSION THREAD

**Discussion by heiler** [iptv-org/community#169] [Ideas]
There are players that do not play stations within a list where the Group Tag is "". 
Would it be an idea, if you file something like "Others" in those tags?
Kind regards

**Comment by fershady** [comment]
Unclassified, 
Not classified, 
Nonclassified 
Uncategorized.

**Comment by what-a** [comment]
Out of curiosity, which IPTV players are unable to play a station due to group-title=""?

**Comment by heiler** [comment]
Hypnotix for example does not.

**Reply by Sphinxroot** [reply]
maybe they should implemented it  , as most player does

**Comment by freearhey** [comment]
Done via #6061

**Comment by heiler** [comment]
When will this solution be effective for playlists by region?

**Reply by freearhey** [reply]
Which playlist exactly are you talking about?

**Comment by heiler** [comment]
I refer to 
https://iptv-org.github.io/iptv/regions/lac.m3u.

**Reply by freearhey** [reply]
Thanks for letting me know about it! I'll try to fix it when I have time.

**Reply by freearhey** [reply]
Fixed via #6439
